modules/apps/transfer/ibc_module.go

```diff
@@ -194,6 +194,7 @@ func (im IBCModule) OnRecvPacket(
 		sdk.NewAttribute(types.AttributeKeyReceiver, data.Receiver),
 		sdk.NewAttribute(types.AttributeKeyDenom, data.Denom),
 		sdk.NewAttribute(types.AttributeKeyAmount, data.Amount),
+		sdk.NewAttribute(types.AttributeKeyMetadata, string(data.Metadata)),
```

Comment:
We may have to emit this in hex format? I believe we ran. into this issue before with the packet data:

https://github.com/cosmos/ibc-go/blob/33639176c35fa421c4affbbc26f7f6fab10cdfe3/modules/core/04-channel/keeper/events.go#L135

https://github.com/cosmos/ibc-go/pull/144

Reply:
fixed